src/services/Service.java

```diff
@@ -0,0 +1,5 @@
+package services;
+
+abstract class Service {
```

Comment:
why this class since is an empty one? the child class ExchangeRateService should not extend this one. what you should do: 
- ExchangeRateService - interface in which you just declare the abstract methods
- ExchangeRateServiceImpl - service class that implements the above interface
- methods from this service should be called in Main not in entity classes 
- each big entity that has business logic should have its own service that holds that logic and whenever you need to call specific actions of the application and to manipulate objects and their state you will do this through their services's methods in Main

Reply:
Aici m-am gandit sa am o implementare in care pot grupa (daca e nevoie mai tarziu) mai multe servicii diferite intr-un arraylist. Adica defineam un list de elemente de tip Service in care adaugam diferite servicii oferite de banca (in cazul in care ma decideam sa implementez mai multe servicii),